### DISCUSSION TITLE
Livestreams in Shorts

### DISCUSSION THREAD

**Discussion by zeroq9** [ReVanced/revanced-discussions#1108] [General] [answered]
Livestreams appear in shorts and when using Autoplay for shorts, it hinders the autoplay because it can't be skipped without scrolling, is there a way to not have livestreams in Shorts or autoskip them after "x" amount of seconds? 

![Screenshot_2024-12-04-15-21-27-43_64ef5fc2000c1caa954c114bb372e1d5.jpg](https://github.com/user-attachments/assets/ae59537d-248b-4677-bd6e-df2e7d7b4ad8)

![Screenshot_2024-12-04-15-18-10-05_64ef5fc2000c1caa954c114bb372e1d5.jpg](https://github.com/user-attachments/assets/69ad4d53-9cf2-4cc6-8b8f-55cc4e7b23f5)

**Comment by LisoUseInAIKyrios** [comment, marked as answer]
Currently there is no way to hide or otherwise filter out livestream Shorts.

See https://github.com/orgs/ReVanced/discussions/1041
